### PR TITLE
added acknowledgement page

### PR DESCRIPTION
**Pull request by spheex**
I added the softwares I know that were used, but I do not know how this page should be organized. @DrDap 
And if other softwares were used? @Samthos

### CODE REVIEW

File: source/acknowledgement/acknowledgement.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Acknowledgement
+===============
+
+©QSimulate acknowledges the use of the following softwares:
+
+-  ATAT
+-  QuantumEspresso
+-  RDKit
+-  Crest
```
Comment (Samthos) [inline source/acknowledgement/acknowledgement.rst:16]: No crest right now

**Review by Samthos — CHANGES_REQUESTED** [review]
We also use pygsm and geometric.

**Comment by spheex** [comment]
Should we mention AWS as the Cloud provider in the Acknowledgement?

**Comment by Samthos** [comment]
@DrDap? This get's into companies acknowledging companies. I know that can be blocked sometimes.

**Pull request merged**